build/Build.cs

```diff
@@ -108,7 +108,7 @@ ITargetDefinition CompileDefinition(ITargetDefinition targetDefinition, [CanBeNu
     Target TestWindowsNet48 => _ => TestDefinition(_, CompileNet48, "net48", runDotMemoryTests: true);
 
     [PublicAPI]
-    Target TestWindowsNet60 => _ => TestDefinition(_, CompileNet60, "net6.0", runDotMemoryTests: true);
+    Target TestWindowsNet80 => _ => TestDefinition(_, CompileNet80, "net8.0", runDotMemoryTests: true);
```

Comment:
I think we would want to keep build net6.0 and test net6.0 and then add the net8.0 versions.